libraries/standard/json/include/json.h

```diff
@@ -70,7 +70,7 @@ JSONStatus_t JSON_Validate( const char * buf,
  * may descend through nested objects when the queryKey contains matching
  * key strings joined by a separator.
  *
- * For example, if buf contains '{"foo":"abc","bar":{"foo","xyz"}}', then a 
+ * For example, if buf contains '{"foo":"abc","bar":{"foo","xyz"}}', then a
```

Comment:
Is {"foo":"abc","bar":{"foo","xyz"}} valid JSON?
Did you mean {"foo":"abc","bar":{"foo":"xyz"}}?

Reply:
Or maybe {"foo":"abc","bar":["foo","xyz"]}

Reply:
Oops.  Fixed.  Thank you!